src/main/java/net/sf/jabref/gui/entryeditor/PdfCommentsTab.java

```diff
@@ -0,0 +1,113 @@
+package net.sf.jabref.gui.entryeditor;
+
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Optional;
+
+import javax.swing.DefaultListModel;
+import javax.swing.JLabel;
+import javax.swing.JList;
+import javax.swing.JPanel;
+import javax.swing.JScrollPane;
+import javax.swing.JTextArea;
+
+import net.sf.jabref.gui.BasePanel;
+import net.sf.jabref.gui.JabRefFrame;
+import net.sf.jabref.logic.pdf.PdfCommentImporter;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.FieldName;
+
+public class PdfCommentsTab extends JPanel {
+
+    private JPanel informationPanel = new JPanel();
+
```

Comment:
Please remove the empty lines between the variable declarations or better group them (e.g. all GUI/panel related stuff first) then a new line and next group.